src/olympia/constants/base.py

```diff
@@ -524,3 +524,15 @@
 
 # Android only started to support WebExtensions with version 48
 DEFAULT_WEBEXT_MIN_VERSION_ANDROID = '48.0'
+
+E10S_UNKNOWN = 0
+E10S_COMPATIBLE = 1
+E10S_COMPATIBLE_WEBEXTENSION = 2
+
+E10S_COMPATIBILITY_CHOICES = (
+    (E10S_UNKNOWN, _('Unknown')),
+    # We don't need to show developers the actual, more granular state, only
+    # that it's compatible or not.
+    (E10S_COMPATIBLE_WEBEXTENSION, _('Compatible')),
```

Comment:
The data is useful for 2 reasons:
- For the developers, to incite them to fix their non-compatible extensions
- For us, to make stats (like arewee10syet, only more current, easier to update data)

It may be a bit overkill to store a separate flag - I for one advocated using a simple boolean in the issue - but it will make queries easier for us later, and adds flexibility.